config/checkstyle_non_main_files_suppressions.xml

```diff
@@ -11,6 +11,7 @@
 
     <!-- Intentional no new line at the end. -->
      <suppress checks="NewlineAtEndOfFile" files="[\\/]test[\\/].*[\\/]grammars[\\/]comments[\\/]InputFullOfSinglelineComments\.java"/>
+     <suppress checks="NewlineAtEndOfFile" files="[\\/]test[\\/].*[\\/]astprinter[\\/]InputFullOfSinglelineComments\.java"/>
```

Comment:
The reason is `AllSinglelineCommentsTest` still uses the input file in `testAllBlockComments()`.